src/prepack-cli.js

```diff
@@ -299,10 +300,10 @@ function run(
           break;
         case "debugDiagnosticSeverity":
           arg = args.shift();
-          invariant(
-            arg === "FatalError" || arg === "RecoverableError" || arg === "Warning" || arg === "Information",
-            `Invalid debugger diagnostic severity: ${arg}`
-          );
+          if (!DiagnosticSeverityValues.includes(arg)) {
```

Comment:
Flow does not understand this test well enough to be know that the assignment in line 307 below is type safe. You can fix that by moving the deleted invariant to just before line 307.